src/main/resources/templates/subscription.html

```diff
@@ -20,53 +20,52 @@
 
     <script type="text/javascript" src="resources/subscription_templates.js"></script>
 
-    <div style="max-width: 100%;" class="container pull-left">
-        <div class="d-flex">
-            <div class="col-lg-10 col-md-10 col-9">
-                <h1 style="font-size:20pt">Eiffel Intelligence Subscription Handling (<span th:text="${frontendServiceUrl}"></span>)
-                </h1>
-            </div>
-            <div class="col-lg-2 col-md-2 col-3">
-                <button data-toggle="tooltip" title="EI connection status check is refreshed continuesly. Click button to force check status."
-                    type="button" id="btnEIConnection" class="btn btnEIConnectionStatus white-space-normal">
-                    <i class="glyphicon"></i>EI Backend Status
-                </button>
+    <div class="d-flex">
+        <div class="col-md-9 col-8">
+            <h3>Eiffel Intelligence Subscription Handling</h3>
+        </div>
+        <div class="col-md-3 col-4 justify-content-end">
+            <div id="btnEIContainer">
+	            <button data-toggle="tooltip" title="EI connection status check is refreshed continuesly. Click button to force check status."
```

Comment:
continuesly is misspelles. should be "continuously".

Reply:
I've corrected that misspelling and added better description of the changes at the top of this pull request.